docs/index.html

```diff
@@ -357,6 +367,68 @@ <h5>Browser extensions</h5>
 
+    <h5>SSH Agent</h5>
+    <dl>
+        <dt id="faq-ssh-agent-how"><a href="#faq-ssh-agent-how">How does the SSH Agent work?</a></dt>
+        <dd>
+            The SSH Agent feature is supported on all target platforms (Linux, macOS and Windows) and it acts as a client
+            for an existing agent despite its name. It can automatically add SSH keys from your KeePassXC database to a
+            running SSH agent when unlocked and remove when locked.<br>
+            <br>
+            On Linux, most desktops are already running an agent without any set up required.<br>
+            On Windows, you need to have <i>Pageant</i> running. It is part of the
+            <a href="https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html">PuTTY suite</a>.<br>
+            On macOS, <code>ssh-agent</code> is running by default and no further setup is required.
+        </dd>
+    </dl>
+    <dl>
+        <dt id="faq-ssh-agent-keys"><a href="#faq-ssh-agent-keys">What SSH key types are supported?</a></dt>
+        <dd>
+            Most major key types are supported (DSA, RSA and Ed25519), including encrypted keys. ECDSA keys are only
+            supported in new OpenSSH file format. Really old 3DES encrypted keys are not and such keys are encouraged
+            to be upgraded for external storage or decrypted for in-database use.<br>
+            <br>
+            PuTTY format key files (.ppk) are <strong>not</strong> supported. You can use <em>PuTTY Key Generator
+            (puttygen.exe)</em> to convert your keys to OpenSSH format.<br>
+            RFC4716 format key files are <strong>not</strong> supported.
+        </dd>
+    </dl>
+    <dl>
+        <dt id="faq-ssh-agent-not-working"><a href="#faq-ssh-agent-not-working">Why are the agent buttons greyed out / why doesn't it work?</a></dt>
+        <dd>
+            On Linux or macOS, you need to have <code>ssh-agent</code> running and <em>SSH_AUTH_SOCK</em> environment
+            variable available for KeePassXC at launch.
+            <a href="https://wiki.archlinux.org/index.php/SSH_keys#SSH_agents">Arch Linux wiki</a> has a generic guide
+            how to manually run <code>ssh-agent</code> if it's not already set up. Sometimes other applications like
+            <em>Gnome Keyring</em> or <code>gpg-agent</code> already provide a compatible agent that also works with
+            KeePassXC.<br>
+            On Windows, Pageant needs to be running, see <a href="#faq-ssh-agent-how">How does the SSH Agent work?</a>.
+        </dd>
+    </dl>
+    <dl>
+        <dt id="faq-ssh-agent-passphrase"><a href="#faq-ssh-agent-passphrase">How do I set up a passphrase for encrypted keys?</a></dt>
+        <dd>
+            The SSH Agent feature uses the entry password field as the decryption key.
+        </dd>
+    </dl>
+    <dl>
+        <dt id="faq-ssh-agent-comment"><a href="#faq-ssh-agent-comment">Why does the public key have no comment or the comment is unexpected?</a></dt>
```

Comment:
Unexpected? Why?

Reply:
Not everyone who generate an Ed25519 possibly understand the base64 encoding also includes the comment and you can't just modify it with a text editor - hence it could be unexpected that is has an embedded comment.

Reply:
Maybe emphasize more that "unexpected" is an error message that a parser can give you? It feels strangely out of context otherwise.

Reply:
It's not a parser error, it's just unexpected for the user.

Reply:
Hmm. Then I would maybe write something along the lines of "Why does the public key (seem) have no comment?"
With your explanation, the "unexpected" makes sense now, but without I wouldn't have know how to interpret it.

Reply:
I'll just remove the unexpected part, maybe no one unexpects the embedded comment.

Reply:
That's a solution, too. :-)